css-view-transitions-2/Overview.bs

```diff
@@ -206,6 +209,83 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 		```
 	</div>
 
+### Example of using 'view-transition-class' ### {#vt-class-example}
+
+In addition to cross-document view-transitions, this document specifies a way to use the same view-transition style
+for multiple elements without having to replicate the corresponding pseudo-elements.
+
+	<div class="example">
+	This example creates a transition with each box participating under its own name, while applying
+	a 1-second duration to the animation of all the boxes:
+
+	```html
+	<div class="box" id="red-box"></div>
+	<div class="box" id="green-box"></div>
+	<div class="box" id="yellow-box"></div>
+	```
+
+	```css
+	div.box {
+		view-transition-class: any-box;
+		width: 100px;
+		height: 100px;
+	}
+	#red-box {
+		view-transition-name: red-box;
+		background: red;
+	}
+	#green-box {
+		view-transition-name: green-box;
+		background: green;
+	}
+	#yellow-box {
+		view-transition-name: yellow-box;
+		background: yellow;
+	}
+
+	/* The following style would apply to all the boxes, thanks to 'view-transition-class' */
+	::view-transition-group(*.any-box) {
+		animation-duration: 1s;
+	}
+	```
+
+	</div>
+
+
+# CSS Properties # {#css-properties}
+
+## Applying the same style to multiple participating elements: the 'view-transition-class' property ## {#view-transition-class-prop}
+
+	<pre class=propdef>
+	Name: view-transition-class
+	Value: none | <<custom-ident>>*
+	Initial: none
+	Inherited: no
+	Percentages: n/a
+	Computed Value: as specified
+	Animation type: discrete
+	</pre>
+
+	The 'view-transition-class' works alongside 'view-transition-name', and is read at the same time
+	'view-transition-name' is read. But unlike 'view-transition-name', 'view-transition-class' doesn't
+	have to be unique - an element can have several view-transition classes, and the same 'view-transition-class'
+	can apply to multiple elements. While 'view-transition-name' is used to match between the element
+	in the old state with its corresponding element in the new state, 'view-transition-class' is used
+	only to apply styles using the view-transition pseudo-elements
+	(''::view-transition-group()'', ''::view-transition-image-pair()'', ''::view-transition-old()'', ''::view-transition-new()'').
+
+	<dl dfn-type=value dfn-for=view-transition-class>
+		: <dfn>none</dfn>
+		:: No class would apply to the [=/element=], and pseudo-elements would have to match its 'view-transition-name'.
+
+		: <dfn><<custom-ident>>*</dfn>
+		:: All of the specified <<custom-ident>> values (apart from <css>none</css>) apply as classes for the [=/element=].
```

Comment:
I don't think we should... I don't see how class being auto would be a useful reserved keyword since the class doesn't affect anything we do, just allow more selection